src/components/Blankslate/Blankslate.tsx

```diff
@@ -32,8 +33,8 @@ export const Blankslate: FunctionComponent<BlankslateProps> = ({
 				</div>
 			</Spacer>
 		)}
-		<h4 className="c-h4">{title}</h4>
-		<p className="c-body-1">{body}</p>
+		<h4 className="c-h4">{ReactHtmlParser(title)}</h4>
```

Comment:
Ik zou gwn `dangerouslySetInnerHTML` doen en het parsen/sanitizen in de client doen.
Daar gebruiken we toch al de sanitize-html package en kunnen we zelf beslissen welke html tags we toelaten.